desktop/conf.dist/hue.ini

```diff
@@ -438,6 +438,12 @@ idle_session_timeout=-1
 # Endpoint to fetch the public key from verification server.
 ## key_server_url=https://ext_authz:8000
 
+# The identifier of the service issued the JWT
```

Comment:
nit: trailing . (consistency across the configs)

and above too, in conf.py to

Reply:
fixed